lib/sla.js

```diff
@@ -65,16 +91,29 @@ function addChannelsToBridge(channel, bridge) {
  *   addChannelsToBridge().
  * @return {Q} Q promise object.
  */
-function createChannel(client, bridge) {
```

Comment:
I don't think createChannel is a very good name for this function anymore since you are doing a lot more than just originating a channel. The fact that you are passing in an existing channel also makes the name very confusing.